.travis.yml

```diff
@@ -3,6 +3,12 @@ rvm:
   - 2.2
 before_script:
   - gem install awesome_bot
+  - npm install doctoc
```

Comment:
Add a package.json file and add doctoc to it, and specify the new dep for anchor-markdown-header there? seems a bit cleaner than using these three commands, but that may just be me...